helm-chart/metabase/values.yaml

```diff
@@ -0,0 +1,228 @@
+replicaCount: 1
+
+image:
+  repository: metabase/metabase
+  pullPolicy: IfNotPresent
+  command: []
```

Comment:
```suggestion
```
Property `image.command` is not used in the chart so can be deleted. Also I'm pretty sure that `command` shouldn't be nested property of the `image` property.

Reply:
Hi there! the command key should be populated with the commands that metabase JAR file exposes, which are many

Reply:
Sorry if my comment is not so clear, but actually there are two comments.

1. I know that key/property `command` is used to override default command for the container in Helm charts. But I don't see that `.Values.image.command` is used in the https://github.com/metabase/metabase/blob/helm-chart/helm-chart/metabase/templates/deployment.yaml  So the property `image.command` should to be deleted from `values.yaml` or it should be used in `deployment.yaml`

2. Usually key/property `command` is placed not under `image` key but as a root key or as a nested key for a component i.e. instead of
```
image:
  repository: metabase/metabase
  pullPolicy: IfNotPresent
  command: []
  tag: "latest"
```
should be
```
image:
  repository: metabase/metabase
  pullPolicy: IfNotPresent
  tag: "latest"

command: []
```